.github/workflows/build.yml

```diff
@@ -191,6 +191,11 @@ jobs:
         shell: bash
         run: |
           make build-wasmer
+      - name: Build Wasmer binary on Wasm32-WASI without LLVM
+        if: matrix.use_llvm != true
```

Comment:
I'm not sure this is the right flag, we may need a new flag which is `build_wasm`